tools/sgcloudspanner/emulator.go

```diff
@@ -107,6 +118,48 @@ func inspectPortAddress(ctx context.Context, containerID, containerPort string)
 	return "", fmt.Errorf("no mapping found for %s in container %s", containerPort, containerID)
 }
 
+func inspectPortAddressCloudbuild(ctx context.Context, containerID, containerPort, image string) (string, error) {
+	var stdout bytes.Buffer
+	cmd := sgdocker.Command(ctx, "inspect", containerID)
+	cmd.Stdout = &stdout
+	if err := cmd.Run(); err != nil {
+		return "", err
+	}
+	var containers []struct {
+		Config struct {
+			Image string
+		}
+		NetworkSettings struct {
+			Ports map[string][]struct {
+				HostIP   string
+				HostPort string
+			}
+			Networks map[string]struct {
+				Gateway string
+			}
+		}
+	}
+	if err := json.NewDecoder(&stdout).Decode(&containers); err != nil {
+		return "", err
+	}
+	fmt.Printf("containers:\n%+v", containers)
+	var host string
+	var port string
+
+	for _, container := range containers {
+		if container.Config.Image == image {
+			port = container.NetworkSettings.Ports[containerPort][0].HostPort // prefer first option
+		}
+		if network, ok := container.NetworkSettings.Networks["cloudbuild"]; ok {
```

Comment:
can we break out `cloudbuild` to a constant? used in three places.

Reply:
Yes, done.